Core/clim-core/presentations/presentation-type-functions.lisp

```diff
@@ -82,18 +82,29 @@ otherwise return false."
 (define-presentation-generic-function %presentation-typep presentation-typep
```

Comment:
Should the commit message say `presentation-typep` instead of `presentation-subtypep`? The latter doesn't seem to be changed in any way.

Reply:
yes, typep. fixed